docs/openstack_backend.rst

```diff
@@ -63,6 +63,10 @@ Usage
     $ openstack keypair create myself > myself.pem
     $ chmod 600 myself.pem
```

Comment:
This bullet point duplicates the last bullet point of the previous section (`Setup`). @toabctl Could you drop it?

Reply:
Done.

Reply:
That section is better to replace with instructions of key creation using `ssh-keygen -m PEM` instead of `openstack keypair create` since newer versions of openstack can create keys with newer format, which is not supported by paramiko at the moment.

Reply:
Which format is not supported?

Reply:
@toabctl the one having "BEGIN OPENSSH PRIVATE KEY"

Reply:
see details here https://wiki.sepia.ceph.com/doku.php?id=testnodeaccess#sshexceptionerror_reading_ssh_protocol_banner

Reply:
Would it be possible to get this PR merged and fix stuff that is unrelated to this PR later? Imo this PR is an improvement over the current state.